### DISCUSSION TITLE
[New Feature] Increase available waypoints from 12 to 32.

### DISCUSSION THREAD

**Discussion by CCHyper** [Vinifera-Developers/Vinifera#1470] [Feature Requests]
### Description:
Currently, the game only allows up to 12 individual waypoint paths. Increasing this limit to 32 _(a technical limit)_ would provide players with more freedom when planning waypoints.

**Comment by Bittah** [comment]
Most players would be fine with just having repeating colors for waypoint paths after the 12th, but if we still want to have a way to differentiate between separate waypoint paths of the same color, all waypoint paths that repeat a color could have a letter added in front of their number (like A1, A2, etc.).

**Comment by CCHyper** [comment]
Reference image for visual purposes;
![image](https://user-images.githubusercontent.com/73803386/122306216-f5c14b00-ceff-11eb-943c-9b8ea499b3ed.png)
